src/main/java/core/basesyntax/service/impl/FruitServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.OperationsStrategy;
+import java.util.List;
+
+public class FruitServiceImpl implements FruitService {
+    @Override
+    public void getAllOperationsStrategy(List<FruitTransaction> fruitFromCsvRow,
```

Comment:
1. rename this method to process or processTransactions
2. pass only List<FruitTransaction> to this method
3. make a fruitStrategy field for this service and pass it in the constructor.